pkg/policy/k8s/cilium_cidr_group_test.go

```diff
@@ -266,6 +266,96 @@ func TestHasCIDRGroupRef(t *testing.T) {
 			cidrGroup: "cidr-group-2",
 			expected:  true,
 		},
+		{
+			name: "CIDR in Spec (Egress)",
+			cnp: &types.SlimCNP{
+				CiliumNetworkPolicy: &cilium_v2.CiliumNetworkPolicy{
+					TypeMeta: metav1.TypeMeta{
+						APIVersion: "cilium.io/v2",
+						Kind:       "CiliumNetworkPolicy",
+					},
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-policy",
+						Namespace: "test-namespace",
+					},
+					Spec: &api.Rule{
+						Ingress: []api.IngressRule{
+							{
+								IngressCommonRule: api.IngressCommonRule{
+									FromCIDRSet: api.CIDRRuleSlice{
+										{
+											CIDRGroupRef: "cidr-group-1",
+										},
+									},
+								},
+							},
+						},
+					},
+					Specs: api.Rules{
+						{
+							Ingress: []api.IngressRule{
+								{
+									IngressCommonRule: api.IngressCommonRule{
+										FromCIDRSet: api.CIDRRuleSlice{
+											{
+												Cidr: "1.1.1.1/32",
+											},
+										},
+									},
+								},
+							},
+						},
+					},
```

Comment:
I think we can remove the `Specs` part for this test case, since we are testing it in the next one.

Reply:
Done.